code/modules/mob/living/carbon/human/life.dm

```diff
@@ -681,6 +681,7 @@
 				update_mutations(0)
 				update_inv_w_uniform(0)
 				update_inv_wear_suit()
+				update_body(0)

```

Comment:
This comes BEFORE the `update_whatever` that doesn't have 0 for an argument - otherwise, `update_icons` never gets called.